app/controllers/application_controller.rb

```diff
@@ -3,20 +3,19 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
-  before_action :check_session_expiry, :set_session_expiry, :set_locale
+  before_action :authenticate_user!
+  before_action :set_locale
   after_action :save_current_store
 
+  def default_url_options
```

Comment:
You seem to have a duplicate method here ?  unless github is showing me this wrong, it is also defined on line 21 ?  Line 21 will be overwriting this